.github/workflows/release.yml

```diff
@@ -43,14 +45,16 @@ jobs:
       - name: Package release
         id: package_release
         run: |
-          VERSION=`cat semver.txt`
+          dotnet tool restore
+          VERSION=`nbgv get-version --variable Version`
+          NUGETVERSION=`nbgv get-version --variable NuGetPackageVersion`
           OUTPUT=./nupkgs
           echo "##[set-output name=version;]$VERSION"
-          echo "##[set-output name=package_name;]CycloneDX.$VERSION.nupkg"
-          echo "##[set-output name=package_filename;]$OUTPUT/CycloneDX.$VERSION.nupkg"
+          echo "##[set-output name=package_name;]CycloneDX.$NUGETVERSION.nupkg"
+          echo "##[set-output name=package_filename;]$OUTPUT/CycloneDX.$NUGETVERSION.nupkg"
           REPO=cyclonedx/cyclonedx-dotnet
-          dotnet build --configuration Release /p:Version=$VERSION
-          dotnet pack CycloneDX/CycloneDX.csproj --configuration Release /p:Version=$VERSION --output $OUTPUT
+          dotnet build --configuration Release
+          dotnet pack CycloneDX/CycloneDX.csproj --configuration Release --output $OUTPUT
```

Comment:
The file with 3 semver version in the filename is created.
```
Successfully created package '/home/runner/work/cyclonedx-dotnet/cyclonedx-dotnet/nupkgs/CycloneDX.3.0.6.nupkg'.
```

Reply:
The version for nuget package uses  “X.X.X” and the assembly version uses “X.X.X.X”.  This was also schema was also used before.

<img width="352" alt="image" src="https://github.com/CycloneDX/cyclonedx-dotnet/assets/2315215/9ea644f7-662e-434e-bb8d-c5061ae48596">